src/ui/React/CharacterOverview.tsx

```diff
@@ -118,6 +118,16 @@ export function Val({ name, color }: ValProps): React.ReactElement {
     return clearSubscription;
   }, [name]);
 
+  if (name === "Int" && Player.bitNodeOptions.intelligenceOverride !== undefined) {
+    return (
+      <Tooltip title={`Intelligence Override: ${formatSkill(Player.bitNodeOptions.intelligenceOverride)}`}>
```

Comment:
Optional: I think the display of overridden int should be a bit more prominent. Maybe something like "X (Y)", where X is the overridden value and Y is the real value, but only if Y > X. I could see that having issues with fitting in the stats overview, though. 

I do think the overridden value is the more important one to show, since it is what will be used for calculations. But again, only if Y > X.

If this is too complicated/annoying, it can be deferred to a follow-up. The int feature is complicated, and I don't want it to block the overall PR more than it already has.

Reply:
I made a small change based on your suggestion. I'll make further changes in a follow-up PR if needed.